content/posts/2021/01/2021-11-04-Getting-Started-With-React/index.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Gettting Started with React
+author: Valary Uhuru
+date: 2021-11-04
+hero: ./images/image.png
+excerpt: This blog post shows step by step guidelines on how to get started with react and resources to use to help one get in track.
+---
+
+
+I can almost bet on it that you've fallen in love with JavaScript and want to dive deeper into learning some of it's libaries and frameworks. Well if that's the case then this is the place to be. This guide will walk you through everything you need to know to get started with React.
+
+We'll get set up, explain the how's and why's behind the basic concepts and build a small project that displays data from an API. I am assuming that we have a little knowledge of JavaScript core concepts such as Asychronous programing, Promises, Hoisting, ES6 Syntax etc. If not visit here [<span style="color:blue"> **JavaScript Concepts That Every Developer Must Know** </span>]( https://www.geeksforgeeks.org/7-javascript-concepts-that-every-developer-must-know/) to learn more.
+
+Grab a drink, buckle up and let's get started!
+
+### Sections
```

Comment:
These section links don't work and render funny.

Suggested fix (example)
 - [Prerequisites](#Prerequisites)
 
Also note the addition of the `ul` marker `-` to add lists

![Screenshot 2021-11-28 at 23 00 02](https://user-images.githubusercontent.com/9129843/143783945-3992afeb-1a41-4ac3-ac4c-a1cda8f18f57.png)